qemu_v8.mk

```diff
@@ -0,0 +1,278 @@
+#        FVP.
+edk2:
+ifeq ("$(wildcard $(EDK2_BIN))","")
+	mkdir -p $(EDK2_PATH)
+	wget -O $(EDK2_BIN) \
+		http://snapshots.linaro.org/components/kernel/leg-virt-tianocore-edk2-upstream/989/QEMU-KERNEL-AARCH64/RELEASE_GCC49/QEMU_EFI.fd
+endif
+	mkdir -p $(ARM_TF_PATH)/build/qemu/release
+	ln -sf $(OPTEE_OS_BIN) $(ARM_TF_PATH)/build/qemu/release/bl32.bin
+	ln -sf $(EDK2_BIN) $(ARM_TF_PATH)/build/qemu/release/bl33.bin
+
+################################################################################
+# QEMU
+################################################################################
+qemu:
+	cd $(QEMU_PATH); ./configure --target-list=aarch64-softmmu --cc="$(CCACHE)gcc" --extra-cflags="-Wno-error"
+	$(MAKE) -C $(QEMU_PATH)
+
+qemu-clean:
+	$(MAKE) -C $(QEMU_PATH) distclean
+
+################################################################################
+# Busybox
+################################################################################
+BUSYBOX_COMMON_TARGET = fvp
+BUSYBOX_CLEAN_COMMON_TARGET = fvp clean
+BUSYBOX_COMMON_CCDIR = $(AARCH64_PATH)
+
+busybox: busybox-common
+
+busybox-clean: busybox-clean-common
+
+busybox-cleaner: busybox-cleaner-common
+
+################################################################################
+# Linux kernel
+################################################################################
+LINUX_DEFCONFIG_COMMON_ARCH := arm64
+LINUX_DEFCONFIG_COMMON_FILES := \
+		$(LINUX_PATH)/arch/arm64/configs/defconfig \
+		$(CURDIR)/kconfigs/qemu.conf
+
+linux-defconfig: $(LINUX_PATH)/.config
+
+LINUX_COMMON_FLAGS += ARCH=arm64
+
+linux: linux-common
+
+linux-defconfig-clean: linux-defconfig-clean-common
+
+LINUX_CLEAN_COMMON_FLAGS += ARCH=arm64
+
+linux-clean: linux-clean-common
+
+LINUX_CLEANER_COMMON_FLAGS += ARCH=arm64
+
+linux-cleaner: linux-cleaner-common
+
+################################################################################
+# OP-TEE
+################################################################################
+OPTEE_OS_COMMON_FLAGS += PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y \
+			 DEBUG=0 CFG_PM_DEBUG=0 CFG_TEE_CORE_LOG_LEVEL=3
+optee-os: optee-os-common
+
+OPTEE_OS_CLEAN_COMMON_FLAGS += PLATFORM=vexpress-qemu_armv8a
+optee-os-clean: optee-os-clean-common
+
+optee-client: optee-client-common
+
+optee-client-clean: optee-client-clean-common
+
+################################################################################
+# Soc-term
+################################################################################
+soc-term:
+	$(MAKE) -C $(SOC_TERM_PATH)
+
+soc-term-clean:
+	$(MAKE) -C $(SOC_TERM_PATH) clean
+
+################################################################################
+# xtest / optee_test
+################################################################################
+XTEST_COMMON_FLAGS += CFG_ARM32=y
+xtest: xtest-common
+
+XTEST_CLEAN_COMMON_FLAGS += CFG_ARM32=y
+xtest-clean: xtest-clean-common
+
+XTEST_PATCH_COMMON_FLAGS += CFG_ARM32=y
```

Comment:
Good question, I probably forgot to change to 64 when I created the file. 64 would be a better choice here, I agree with that.